src/core/interaction_data.cpp

```diff
@@ -585,19 +585,30 @@ void set_dipolar_method_local(DipolarInteraction method) {
 /*                                 electrostatics */
 /********************************************************************************/
 
-int coulomb_set_bjerrum(double bjerrum) {
-  if (bjerrum < 0.0)
+int coulomb_set_prefactor(double prefactor)
+{
+  if (prefactor < 0.0) {
```

Comment:
Do we really want to have that limitation? At least the gpu p3m works fine with negative prefactor (=gravity).